### PR TITLE
Add VonMisesFeaturizer

### PR DESCRIPTION
**Pull request by cxhernandez**
re: #741 and https://github.com/msmbuilder/msmbuilder/pull/743

### CODE REVIEW

**Comment by cxhernandez** [comment]
@mpharrigan: better?

File: msmbuilder/featurizer/featurizer.py [hunk 1]
```diff
@@ -445,14 +453,115 @@ def partial_transform(self, traj):
...
+                d_i = dict(resname=resnames[i], atomind=aind[i],
+                           resSeq=resSeq[i], resid=resid[i],
+                           otherInfo=otherInfo[i], bigclass=bigclass[i],
+                           smallclass=smallclass[i])
+                x.append(d_i)
+
+        return x
+
+    def partial_transform(self, traj):
+        """Featurize an MD trajectory into a vector space via calculation
```
Comment (mpharrigan) [inline msmbuilder/featurizer/featurizer.py]: update docstring

**Comment by mpharrigan** [comment]
:+1:

**Comment by cxhernandez** [comment]
This test seems to have passed on my local machine...

```
msmbuilder.tests.test_featurizer.test_that_all_featurizers_run ... ok
```

**Comment by cxhernandez** [comment]
Okay to merge?

**Comment by rmcgibbo** [comment]
Can you test something (e.g. that the shape of the output behaves properly when you change n_bins or kappa or something) in the tests?

**Comment by mpharrigan** [comment]
and add it to the changelog

**Comment by cxhernandez** [comment]
Not sure what that appveyor error is, but it looks unrelated

**Pull request merged**